collectors/statsd.plugin/k6.md

```diff
@@ -0,0 +1,72 @@
+<!--
+title: "K6 Load Testing monitoring with Netdata"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/stats.d.plugin/k6.md
+sidebar_label: "K6 Load Testing"
+-->
+
+# K6 Load Testing monitoring with Netdata
+
+Monitors the performance of your Load Testing experiments using [K6](https://k6.io/).
+
+You can read more about the metrics that K6 sends in the [K6 documentation](https://k6.io/docs/using-k6/metrics/). 
+
+## Requirements
+
+-   When running the k6 experiment, specify a [StatsD output](https://k6.io/docs/results-visualization/statsd/).
+    -   Tip: K6 currently supports tags only with [datadog output](https://k6.io/docs/results-visualization/datadog/), which is in essence StatsD. Netdata can be used with both.
+
+
+1. **HTTP Requests**
+
+Number of HTTP requests that K6 generates, per second.
```

Comment:
I think you need to "embed" the text of each of these into the list itself. I don't know if that makes sense, but look here:

```suggestion
1. **HTTP Requests**
   Number of HTTP requests that K6 generates, per second.
```

You'll have to do this for each of the numbered items below.